lib/config/presets/internal/custom-managers.ts

```diff
@@ -90,12 +90,7 @@ export const presets: Record<string, Preset> = {
     customManagers: [
       {
         customType: 'regex',
-        fileMatch: [
-          '(^|/)Makefile$',
-          '(^|/)makefile$',
-          '(^|/)GNUMakefile$',
-          '\\.mk$',
-        ],
+        filePatterns: ['**/Makefile', '**/GNUMakefile', '**/*.mk'],
```

Comment:
are you sure glob is case insensitive?
```suggestion
        filePatterns: ['**/{M,m}akefile', '**/GNUMakefile', '**/*.mk'],
```

Reply:
Rhys, suggested that it is case insensitive. I didn't double check and went with the suggestion.

I looked it up and there seems to be mixed answers. I will apply your suggestion to be 100% safe.